src/client/network.ts

```diff
@@ -161,6 +161,16 @@ export class Request extends ChannelOwner<channels.RequestChannel, channels.Requ
     return (await this._getHeadersIfNeeded()).map(header => [header.name, header.value]);
   }
 
+  async headerValue(name: string): Promise<string | null> {
+    const headers = await this._getHeadersIfNeeded();
```

Comment:
I would prefer to have the same implementation as with headerValue for response.

Reply:
done